core/src/main/java/com/coderanch/blackjack/Hand.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2018 Coderanch.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.coderanch.blackjack;
+
+import java.util.*;
+
+import static com.coderanch.blackjack.Card.Rank.ACE;
+import static com.coderanch.util.require.Require.requireThat;
+import static java.util.function.Predicate.not;
```

Comment:
Sort and group your import statements. Put imports from java packages at the top, regardless of alphabetical order.